iron-doc-behavior.html

```diff
@@ -111,6 +111,25 @@
         return prefix + type + suffix;
       },
 
+      /**
+       * Scrolls to the currently selected anchor, as identified
+       * by the URL hash. Whichever element or script is in charge
+       * of routing should call this method on initial page load and
+       * on hashchange events.
+       */
+      scrollToAnchor: function(hash) {
+        // ToDo: handle linking to private members
+        if (hash && hash.length > 1) {
+          // ensure all dom-repeats have rendered.
+          Polymer.dom.flush();
+          var anchorId = window.location.hash.slice(1);
```

Comment:
Ouch, this was existing code... just happened to work probably because the forwarder was reading from `window.location.hash`. I'd still support reading the global hash, but do it above when the parameter isn't defined. Update the docs too.